app/domain/urls.ts

```diff
@@ -9,7 +9,8 @@ export type Locale =
   | 'it'
   | 'tr'
   | 'dk'
-  | 'es-CO';
+  | 'es-CO'
+  | 'fr-FR'  ;
```

Comment:
Please run `yarn lint` and fix any errors you see

Reply:
It should be ok now!